tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -44,15 +44,7 @@ def c2r_config_setup(request, backup_directory):
     [
         Config(
             "~/.convert2rhel.ini",
-            # We disable the Black pre-commit hook because it changes formatting
-            # of the following configuration files, which makes them hard to read.
-            # https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections
-            # fmt: off
-            (
-                "[subscription_manager]\n"
-                "password = config_password\n"
-            ),
-            # fmt: on
+            ("[subscription_manager]\n" "password = config_password\n"),
```

Comment:
Is there a possibility to disable the ruff for a specific block of code? I would like to keep this to improve the readability. The only thing I found is this https://www.reddit.com/r/neovim/comments/1aeinr5/how_to_make_ruff_ignore_one_code_block/, but it suggests what is already in the code -> to use the `fmt: off`

Reply:
Ah you're right, should probably move this back

https://docs.astral.sh/ruff/formatter/#format-suppression

Reply:
Fixed it  @kokesak

Reply:
Thanks!